hardhat.config.ts

```diff
@@ -10,13 +10,14 @@ dotenv.config();
 const config: HardhatUserConfig = {
     solidity: "0.6.12",
     starknet: {
-        dockerizedVersion: "0.10.3", // alternatively choose one of the two venv options below
+        dockerizedVersion: "0.11.0.1", // alternatively choose one of the two venv options below
         // uses (my-venv) defined by `python -m venv path/to/my-venv`
         // venv: "path/to/my-venv",
 
         // uses the currently active Python environment (hopefully with available Starknet commands!)
         // venv: "active",
         recompile: false,
+        manifestPath: "path/to/Cargo.toml",
```

Comment:
Can this be commented? I've had the practice to leave uncommented those lines which are generally expected to work. And this one isn't since this path doesn't generally exist on users' computers. Compare with the commented `venv: "path/to/my-venv"`.

Reply:
https://github.com/Shard-Labs/starknet-hardhat-example/pull/104/commits/cce42e19f4fdd4be150246cc056180a8f603bb53